janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchSetup.java

```diff
@@ -101,96 +78,6 @@ public Connection connect(Configuration config) throws IOException {
         }
     };
 
-    /**
-     * Build and setup a new ES settings builder by consulting all JanusGraph config options
-     * relevant to TransportClient or Node.  Options may be specific to a single client,
-     * but in this case they have no effect/are ignored on the other client.
-     * <p>
-     * This method creates a new ES ImmutableSettings.Builder, then carries out the following
-     * operations on that settings builder in the listed order:
-     *
-     * <ol>
-     *  <li>Enable client.transport.ignore_cluster_name in the settings builder</li>
-     *  <li>If conf-file is set, open it using a FileInputStream and load its contents into the settings builder</li>
-     *  <li>Apply any settings in the ext.* meta namespace</li>
-     *  <li>If cluster-name is set, copy that value to cluster.name in the settings builder</li>
-     *  <li>If ignore-cluster-name is set, copy that value to client.transport.ignore_cluster_name in the settings builder</li>
-     *  <li>If client-sniff is set, copy that value to client.transport.sniff in the settings builder</li>
-     *  <li>If ttl-interval is set, copy that volue to indices.ttl.interval in the settings builder</li>
-     *  <li>Unconditionally set script.inline to true (i.e. enable inline scripting)</li>
-     * </ol>
-     *
-     * This method then returns the builder.
-     *
-     * @param config a JanusGraph configuration possibly containing Elasticsearch index settings
-     * @return ES settings builder configured according to the {@code config} parameter
-     * @throws java.io.IOException if conf-file was set but could not be read
-     */
-    private static Map<String,Object> settingsBuilder(Configuration config) throws IOException {
-
-        Map<String,Object> settings = new HashMap<>();
-
-        // Set JanusGraph defaults
-        settings.put("client.transport.ignore_cluster_name", true);
-
-        settings.put("path.home", System.getProperty("java.io.tmpdir"));
-
-        // Apply overrides from ES conf file
-        applySettingsFromFile(settings, config, INDEX_CONF_FILE);
-
-        // Apply ext.* overrides from JanusGraph conf file
-        applySettingsFromJanusGraphConf(settings, config, ElasticSearchIndex.ES_EXTRAS_NS);
-
```

Comment:
@sjudeng since this PR, ES_EXTRA_NS is still in ElasticSearchIndex class and in doc. But JanusGraph never use it.
I miss something, or it's had to be removed ?

Reply:
@davidclement90 This is in a private method block (`Map<String,Object> settingsBuilder(Configuration config)`) that had no usages, which is why it was removed. But the [ElasticSearchSetup#applySettingsFromJanusGraphConf](https://github.com/JanusGraph/janusgraph/blob/b9e83bd56dc90d57d4e84c0c96fe649bf6787488/janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchSetup.java#L81-L83) method itself is still there and is used in [ElasticSearchIndex](https://github.com/JanusGraph/janusgraph/blob/b9e83bd56dc90d57d4e84c0c96fe649bf6787488/janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java#L235). But is something breaking for you since the update?

Reply:
For me in ElasticSearchIndex, when we call applySettingsFromJanusGraphConf, we use ES_CREATE_EXTRAS_NS and not ES_EXTRAS_NS. 
ES_EXTRAS_NS is no longer used because Elasticsearch can only be query with Rest so we can not overrides ES yaml settings.
So for me, ES_EXTRAS_NS has to be removed and the doc has to be updated. 
What do you think ?

Reply:
@davidclement90 So you're recommending we confirm ES_EXTRA_NS is no longer supported and then remove it and update docs, right? It makes sense to me.

Reply:
Yes, I grep over all the code and I just find a match for the definition in ElasticSearchIndex and in one comment.
`grep -ri ES_EXTRAS_NS .
./janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchSetup.java: * Any key-value pairs under the {@link org.janusgraph.diskstorage.es.ElasticSearchIndex#ES_EXTRAS_NS} namespace
./janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java:    public static final ConfigNamespace ES_EXTRAS_NS =`

I just open an issue : https://github.com/JanusGraph/janusgraph/issues/545